src/Features/Blockcore.Features.NodeHost/Startup.cs

```diff
@@ -123,7 +123,7 @@ public void ConfigureServices(IServiceCollection services)
                     options.SwaggerDoc("node",
                            new OpenApiInfo
                            {
-                               Title = "Blockcore Node API",
+                               Title = hostSettings.NetworkAgent+" Node API",
```

Comment:
is it set to default somewhere?

Reply:
The default value for NetworkAgent is Blockcore (on NodeSettings.cs L126)